### PR TITLE
[PM-233] "No Items" graphic is overlapping on several screens

### PR DESCRIPTION
**Pull request by cagonzalezcs**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

The graphic used to indicate to users that no items are available within a vault is overlapping the list of items within the vault, and is being displayed when it should not be displayed. Specific issues identified include:
- Displaying the graphic when a parent folder has sub folders, and one of those sub folders contains an item. This happens while the parent folder has no items itself.
- Displaying the graphic when an account has send disabled by an administrator.

## Code changes

- **src/vault/popup/components/vault/vault-items.component.html**: Added and `ngIf` directive to the "no-items" graphic that removes the graphic is the view has either nested folders or nested collections.
- **src/tools/popup/send/send-groupings.component.html**: Added a boolean check to identify if `disableSend` is set to true, if so then the `no-items` graphic will not be displayed. This addresses an issue where the send functionality is disabled for the user.
- **src/popup/scss/base.scss**: Made a CSS change to ensure that background-color does not change on hover for disabled `button` or `a` elements. We are forcing the original background color of the non-hovered element by setting `background-color: inhert` with an important declaration. 

## Screenshots

https://user-images.githubusercontent.com/16629865/235030358-e5cbcb14-c8b7-444f-ab31-f0ef643c4b5e.mov

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/browser/src/tools/popup/send/send-groupings.component.html [hunk 1]
```diff
@@ -12,6 +12,8 @@ <h1 class="sr-only">{{ "send" | i18n }}</h1>
       (input)="search(200)"
       autocomplete="off"
       appAutofocus
+      [readonly]="disableSend"
+      [disabled]="disableSend"
```
Comment (djsmith85) [inline apps/browser/src/tools/popup/send/send-groupings.component.html:16]: Even if a member of an org, due to a policy, is not able to view/edit a Send: I'd still say they'd be able to use the search to find the Send, they want to delete.

**Review by djsmith85 — CHANGES_REQUESTED** [review]
Welcome to the team @cagonzalezcs 🎉 

just one small thing.

File: apps/browser/src/tools/popup/send/send-groupings.component.html (hunk 1 above)
Comment (cagonzalezcs) [reply apps/browser/src/tools/popup/send/send-groupings.component.html:16]: Understood, wasn't sure on whether this should be modifiable or not given the policy within an organization. Will remove this modification from the scope of changes.

Comment (cagonzalezcs) [reply apps/browser/src/tools/popup/send/send-groupings.component.html]: @djsmith85 This has been removed as requested.

**Review by djsmith85 — APPROVED** [review]
team-tools-dev approves

**Review by differsthecat — APPROVED** [review]
Looks good from vault, approved!

**Pull request merged**